src/utils/string/index.ts

```diff
@@ -1,6 +1,8 @@
 import { format, utcToZonedTime } from "date-fns-tz";
 import get from "lodash/get";
 
+export { githubPRLinkify } from "./githubPRLinkify";
```

Comment:
[nit] We could export `jiraLinkify` here too for consistency

Reply:
jiraLinkify actually can't be exported from index without running into a dep cycle. jiraLinkify imports externalResources and the contents of string and externalResources are exported from utils